gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/reactive/reactor/DefaultTcpSocketDispatcher.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.gateway.reactive.reactor;
+
+import static io.gravitee.gateway.reactive.api.context.InternalContextAttributes.ATTR_INTERNAL_LISTENER_TYPE;
+
+import io.gravitee.common.http.IdGenerator;
+import io.gravitee.gateway.core.component.ComponentProvider;
+import io.gravitee.gateway.reactive.api.ListenerType;
+import io.gravitee.gateway.reactive.api.context.InternalContextAttributes;
+import io.gravitee.gateway.reactive.core.context.DefaultExecutionContext;
+import io.gravitee.gateway.reactive.reactor.handler.TcpAcceptorResolver;
+import io.gravitee.gateway.reactor.handler.TcpAcceptor;
+import io.gravitee.gateway.tcp.reactive.VertxTcpRequest;
+import io.gravitee.gateway.tcp.reactive.VertxTcpResponse;
+import io.reactivex.rxjava3.core.Completable;
+import io.vertx.rxjava3.core.net.NetSocket;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @author Benoit BORDIGONI (benoit.bordigoni at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@RequiredArgsConstructor
+@Slf4j
+public class DefaultTcpSocketDispatcher implements TcpSocketDispatcher {
+
+    private final TcpAcceptorResolver tcpAcceptorResolver;
+    private final ComponentProvider componentProvider;
+    private final IdGenerator idGenerator;
+
+    @Override
+    public Completable dispatchSocket(NetSocket proxySocket, String serverId) {
```

Comment:
HttpRequestDispatcher and SubscriptionDispatcher declare a `dispatch(args)` method, so I propose this to be consistent:
```suggestion
    public Completable dispatch(NetSocket proxySocket, String serverId) {
```